contracts/bonding/BondingManager.sol

```diff
@@ -660,9 +664,16 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
             earningsPool.setStake(t.earningsPoolPerRound[lastUpdateRound].totalStake);
         }
 
+        uint256 rewardTokens = 0;
         // Create reward based on active transcoder's stake relative to the total active stake
-        // rewardTokens = (current mintable tokens for the round * active transcoder stake) / total active stake
-        uint256 rewardTokens = minter().createReward(earningsPool.totalStake, currentRoundTotalActiveStake);
+        if (currentRound >= roundsManager().lipUpgradeRound(56)) {
```

Comment:
Do we need this if-else statement ? Reward can't be called for a previous round so the code path for the old approach wouldn't be available after the upgrade anyway (i.e. `currentRound >= roundsManager().lipUpgradeRound(56)` will always evaluate to true after the upgrade took place, no ?)  

The only real reason I can see this being useful is for testing purposes and private deployments?